app/gui/view/graph-editor/src/component/node.rs

```diff
@@ -785,6 +785,8 @@ impl Node {
         frp::extend! { network
             output_hover <- model.output.on_port_hover.map(|s| s.is_on());
             visualization_hover <- bool(&model.visualization.on_event::<mouse::Out>(), &model.visualization.on_event::<mouse::Over>());
+            is_preview <- visualization.view_state.map(|s| s.is_preview());
+            visualization_hover <- visualization_hover.gate(&is_preview);
```

Comment:
I would rename `visualization_hover` to `preview_hover`.

Reply:
Done!